signac/common/config.py

```diff
@@ -88,20 +111,15 @@ def read_config_file(filename):
     return config
 
 
-def get_config(infile=None, configspec=None, * args, **kwargs):
+def get_config(infile=None, configspec=None, *args, **kwargs):
+    """Get configuration from a file."""
```

Comment:
Since I'm not too much friendly with the code for this repository, I'm not sure whether we should describe these params or not.